### ISSUE TITLE
Eagle file downloads

### ISSUE THREAD

**Issue by REHolt** [8n1/ESP8266-Breakout-Adapter#3]
Hi, I was wanting to download your Eagle CAD files to take a look at your circuit. I want to expand on this and build a couple of special purpose boards for the ESP12E that I bought. However, when I download the  files and try to open them in Eagle CAD, I get an error saying they are not eagle files. I am using Eagle ver 7.6.0 (the latest version). yours appear to be 7.4 so they should be compatible. Anything special I need to do to download them?

**Comment by REHolt** [comment]
I found a solution. I opened the file on the github page, in RAW format, I then saved it to my project directory. For some reason Eagle was able to read it that way even though directly saving the file didn't work. 
Thanks for posting this, it will be a big help in getting my ESP board build, and in helping me to learn Eagle CAD. Having your board layout to use as an example will help me get to a solution faster.

My question now is about the bottom row of pins on the ESP12E. On the board layout they are there, but in the symbol they are not. I assume you manually placed pads for this row? I asked because I have not been able to find a proper board symbol for the part. Since I am currently less than half way up the learning curve for Eagle cad, I didn't want to attempt making this symbol myself:)
